charts/mlrun-ce/values.yaml

```diff
@@ -405,3 +405,176 @@ kube-prometheus-stack:
+  # 4. role: currently role is not supported - every node is able to be mnode and vnode.
+  #
+  # Btw, keep quotes "" around the value like below, even the value will be number or not.
+  taoscfg:
+    # Starts as cluster or not, must be 0 or 1.
+    #   0: all pods will start as a seperate TDengine server
+    #   1: pods will start as TDengine server cluster. [default]
+    CLUSTER: "0"
+
+    # number of replications, for cluster only
+    TAOS_REPLICA: "1"
+
+
+    #
+    # TAOS_NUM_OF_RPC_THREADS: number of threads for RPC
+    #TAOS_NUM_OF_RPC_THREADS: "2"
+
+    #
+    # TAOS_NUM_OF_COMMIT_THREADS: number of threads to commit cache data
+    #TAOS_NUM_OF_COMMIT_THREADS: "4"
+
+    # enable/disable installation / usage report
+    #TAOS_TELEMETRY_REPORTING: "1"
+
+    # time interval of system monitor, seconds
+    #TAOS_MONITOR_INTERVAL: "30"
+
+    # time interval of dnode status reporting to mnode, seconds, for cluster only
+    #TAOS_STATUS_INTERVAL: "1"
+
+    # time interval of heart beat from shell to dnode, seconds
+    #TAOS_SHELL_ACTIVITY_TIMER: "3"
+
+    # minimum sliding window time, milli-second
+    #TAOS_MIN_SLIDING_TIME: "10"
+
+    # minimum time window, milli-second
+    #TAOS_MIN_INTERVAL_TIME: "1"
+
+    # the compressed rpc message, option:
+    #  -1 (no compression)
+    #   0 (all message compressed),
+    # > 0 (rpc message body which larger than this value will be compressed)
+    #TAOS_COMPRESS_MSG_SIZE: "-1"
+
+    # max number of connections allowed in dnode
+    #TAOS_MAX_SHELL_CONNS: "50000"
+
+    # stop writing logs when the disk size of the log folder is less than this value
+    #TAOS_MINIMAL_LOG_DIR_G_B: "0.1"
+
+    # stop writing temporary files when the disk size of the tmp folder is less than this value
+    #TAOS_MINIMAL_TMP_DIR_G_B: "0.1"
+
+    # if disk free space is less than this value, taosd service exit directly within startup process
+    #TAOS_MINIMAL_DATA_DIR_G_B: "0.1"
+
+    # One mnode is equal to the number of vnode consumed
+    #TAOS_MNODE_EQUAL_VNODE_NUM: "4"
+
+    # enbale/disable http service
+    #TAOS_HTTP: "1"
+
+    # enable/disable system monitor
+    #TAOS_MONITOR: "1"
+
+    # enable/disable async log
+    #TAOS_ASYNC_LOG: "1"
+
+    #
+    # time of keeping log files, days
+    #TAOS_LOG_KEEP_DAYS: "0"
+
+    # The following parameters are used for debug purpose only.
+    # debugFlag 8 bits mask: FILE-SCREEN-UNUSED-HeartBeat-DUMP-TRACE_WARN-ERROR
+    # 131: output warning and error
+    # 135: output debug, warning and error
+    # 143: output trace, debug, warning and error to log
+    # 199: output debug, warning and error to both screen and file
+    # 207: output trace, debug, warning and error to both screen and file
+    #
+    # debug flag for all log type, take effect when non-zero value\
+    #TAOS_DEBUG_FLAG: "143"
+
+    # generate core file when service crash
+    #TAOS_ENABLE_CORE_FILE: "1
```

Comment:
If all these values are optional and not needed, let's remove them as well and only add what is necessary. This values file is already quite large

Reply:
removed